public/client.js

```diff
@@ -18,10 +35,10 @@ $(function () {
 
 		e.preventDefault();
 		var $form = $(this);
-
+console.log(foodTypes);
```

Comment:
Make sure to remove console.log call before code goes to production. It's not as big of a deal as it used to be, but some older browsers, such as IE7, couldn't run the code because it had no console available unless the developer tools were open. This caused a script error that would break other scripts.